includes/definitions/discovery/ciena-waveserver.yaml

```diff
@@ -0,0 +1,24 @@
+mib: CIENA-WS-XCVR-MIB
+modules:
+    sensors:
+        pre-cache:
+            data:
+                - oid:
+                    - cwsXcvrTemperatureThresholdHighAlarmThreshold
+                    - cwsXcvrCienaIdDescription
+                    - cwsXcvrCienaIdCienaItemNumber
```

Comment:
cwsXcvrCienaIdCienaItemNumber seems unused

Reply:
yep. That's a forgotten line ;)